code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -803,23 +803,27 @@
 
 /obj/item/melee/blood_magic/manipulator/attack_self(mob/living/user)
 	if(iscultist(user))
-		var/list/options = list("Blood Spear (150)", "Blood Bolt Barrage (300)", "Blood Beam (500)")
-		var/choice = input(user, "Choose a greater blood rite...", "Greater Blood Rites") as null|anything in options
-		if(!choice)
+		var/list/spells = list(
```

Comment:
```suggestion
		var/static/list/spells = list(
```
This list won't ever change, right?
Cacheing would be the positive.

Reply:
Noted.